guides/source/active_record_validations.md

```diff
@@ -679,13 +743,14 @@ which would result in a `NULL` value in most cases.
 
 The default error message is _"can't be blank"_.
 
-[`Object#blank?`]: https://api.rubyonrails.org/classes/Object.html#method-i-blank-3F
+[`Object#blank?`]:
+    https://api.rubyonrails.org/classes/Object.html#method-i-blank-3F
 
 ### `absence`
```

Comment:
These validations seems to be sorted alphabetically, except for `absence`.
Ah, it seems `inclusion` and `exclusion` as seperate sections also broke this pattern.
No specific thing to change here I guess, just wanted to make a note.

Reply:
Since `absence` is a standalone section and does not need to be aside `presence` I've moved `absence` to the top of the Validations section.  It also has a link to `presence` which helps `absence` find its partner. See https://github.com/rails/rails/pull/52951/commits/7edd0b809e536008624b8e28ef355f92082c2f1d